app/src/main/res/layout/activity_enter_mnemonic.xml

```diff
@@ -81,20 +82,20 @@
             app:layout_constraintTop_toBottomOf="@+id/inputMnemonicTIL"
             tools:text="Mnemonic" />
 
-        <com.google.android.material.button.MaterialButton
-            android:id="@+id/createWalletFromMnemonicButton"
-            android:layout_width="0dp"
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/subWelcomeTextView"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="16dp"
-            android:layout_marginTop="8dp"
-            android:layout_marginEnd="16dp"
-            android:text="Create wallet from mnemonic"
-            android:textColor="@android:color/white"
-            app:backgroundTint="@color/colorAccent"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="8dp"
+            android:text="Enter your 12 word mnemonic or seed"
```

Comment:
Hard coded string?

Reply:
This screen is old and will be gone/redone